gpcontrib/arenadata_toolkit/sql/arenadata_toolkit_tracking.sql

```diff
@@ -0,0 +1,116 @@
+-- Tests for size tracking logic introduced in version 1.7
+-- start_ignore
+\! gpconfig -c shared_preload_libraries -v 'arenadata_toolkit'
+\! gpstop -raq -M fast
+\! gpconfig -c arenadata_toolkit.tracking_worker_naptime_sec -v '5'
+\! gpstop -raq -M fast
+\c
+-- end_ignore
+-- start_matchsubs
+-- m/ERROR:  database \d+ is not tracked \(track_files\.c:\d+\)/
+-- s/\d+/XXX/g
+-- end_matchsubs
+--start_ignore
+DROP DATABASE IF EXISTS tracking1;
+--end_ignore
+CREATE DATABASE tracking_db1;
+\c tracking_db1;
+CREATE EXTENSION arenadata_toolkit;
+
+-- 1. Test getting track on not registered database;
+SELECT pg_sleep(current_setting('arenadata_toolkit.tracking_worker_naptime_sec')::int);
+SELECT * FROM arenadata_toolkit.tracking_get_track();
+
+SELECT arenadata_toolkit.tracking_register_db();
+
+-- 2. Test initial snapshot behaviour. Triggering initial snapshot leads to
+-- setting up the bloom filter such that all relfilenodes are considered.
+SELECT arenadata_toolkit.tracking_trigger_initial_snapshot();
+SELECT is_triggered FROM arenadata_toolkit.is_initial_snapshot_triggered;
+
+-- 3. If user hasn't registered any schema, the default schemas are used.
+-- See arenadata_toolkit_guc.c. At commit the bloom filter is cleared. The next
+-- call of tracking_get_track() will return nothing if database is not modified in between. 
+SELECT count(*) FROM arenadata_toolkit.tracking_get_track();
+
+-- 4. Create table in specific schema and register that schema.
+CREATE TABLE arenadata_toolkit.tracking_t1 (i INT)
+WITH (appendonly=true, orientation=column) DISTRIBUTED BY (i);
+
+SELECT arenadata_toolkit.tracking_register_schema('arenadata_toolkit');
+
+-- Getting the track. Only created table with size 0 is expected;
+SELECT relname, size, state, segid, relkind, relstorage
+FROM arenadata_toolkit.tracking_get_track();
+
+-- 5. Test data extending event. Bloom should capture it.
+INSERT INTO arenadata_toolkit.tracking_t1 SELECT generate_series(1,100000);
+SELECT relname, size, state, segid, relkind, relstorage
+FROM arenadata_toolkit.tracking_get_track();
+
+-- 6. Dropping table. The track shows only relfilenodes without names and other additional info with status 'd'.
```

Comment:
We are talking about relfilenodes, but do not show them in the query below. I think `relfilenodes` should be added to the target list.

BTW, will it be convenient for the user to receive empty names after table is dropped?